whistle/lib/main.dart

```diff
@@ -1,5 +1,7 @@
 import 'package:flutter/material.dart';
-import 'UploadAudioPage.dart';
```

Comment:
Imo good decision to remove this page, especialluy if it wont have any functionality in teh future/ stashed for future changes